src/zenml/integrations/gcp/orchestrators/vertex_orchestrator.py

```diff
@@ -737,6 +740,11 @@ def _upload_and_run_pipeline(
                 self.config.network,
             )
 
+        if self.config.private_service_connect:
+            run.psc_interface_config = PscInterfaceConfig(
```

Comment:
Are you sure this exist for the `PipelineJob` class that we're using? When browsing my local installation, the installed python code does not seem to have this attribute

Reply:
@stefannica exactly, thats what made this task a bit more complicated. please reference the JIRA ticket for more details if necessary :-)

Reply:
You're right, I had to dig a bit deeper to properly hack this, but I now have something that works. We only need to use this until the v1beta1 code makes it into the stable API.